server/src/app.js

```diff
@@ -3,16 +3,20 @@ const createError = require("http-errors");
 const express = require("express");
 const path = require("path");
 const cookieParser = require("cookie-parser");
-const logger = require("morgan");
+const morgan = require("morgan");
 const helmet = require("helmet");
 const cors = require("cors");
+const logger = require("./logger");
 
 const app = express();
 app.set("query parser", "simple");
 
-process.env.NODE_ENV !== "test" && app.use(logger("dev"));
+if (process.env.NODE_ENV !== "test") {
+  app.use(morgan("dev", { stream: logger.stream }));
```

Comment:
Shouldn't this be `production` rather than `dev`? Or am I misunderstanding it?

Reply:
`dev` here is the logging level, one below `info`. I did think it odd that the `NODE_ENV` was checking `"test"` instead of `"production"`, but decided to leave it as it is until I fully understood why it was set that way.

Reply:
> Duh, of course. I updated the code to use the logger instead of `console.{log,error}` in the listed file.

Ah, I see, because all of the other messages are explicitly `console.log` they would bypass the winston transport functionality? Therefore, only the `logger` reference would be eligible for winston transports.

Reply:
Yeah, exactly.